modules/ossm-adding-project-using-smm-resource-console.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-smm-resource-console_{context}"]
+= Adding a project to the service mesh using the ServiceMeshMember resource with the web console
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* Name of the `ServiceMeshControlPlane` resource and the name of the project that the resource belongs to.
+* Name of the project you want to add to the mesh.
+* A service mesh administrator must explicitly grant access to the service mesh. Administrators can grant users permissions to access the mesh by assigning them the `mesh-user` `Role` using a `RoleBinding` or `ClusterRoleBinding`. For more information, see xref:../../service_mesh/v2x/ossm-profiles-users.html#ossm-members_ossm-profiles-users[Creating the {SMProductName} members].
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project that you want to add to the mesh from the drop-down list. For example `istio-system`.
+
+. Click the {SMProductName} Operator.
+
+. Click the *Istio Service Mesh Member* tab.
+
+. Click *Create ServiceMeshMember*
+
+. Accept the default name for the `ServiceMeshMember`.
+
+. Select the project you want to add to the mesh.
+
+. Click to expand *ControlPlaneRef*. 
+
+. In the *Namespace* field, select the project that the `ServiceMeshControlPlane` resource belongs to. For example, `istio-system`. 
+
+. In the *Name* field, enter the name of the `ServiceMeshControlPlane` resource that this namespace belongs to. For example, `basic`.  
+
+. Click *Create*.
+
+. To confirm the `ServiceMeshMember` resource was created, and that the project was added to the mesh, click the name of the resource (for example, `default`). View the *Conditions* section shown at the bottom of the screen, and confirm that the `Status` of the `Reconciled` and `Ready` conditions is `True`. If the `Status` is `False`, see the `Reason` and `Message` columns for more information.
```

Comment:
```suggestion
. To confirm the `ServiceMeshMember` resource was created, and that the project was added to the mesh, click the resource name; for example, `default`. View the *Conditions* section at the end of the screen, and confirm that the `Status` of the `Reconciled` and `Ready` conditions is `True`. If the `Status` is `False`, see the `Reason` and `Message` columns for more information.
```
**IBM Style**:

-  Accessibility and globalization requirement: People who have limited vision, or who use interfaces that are translated into a bidirectional language, might not understand information if it is conveyed only by location. When you refer to the location of an element in a graphical user interface, do not use sensory words, such as top, bottom, right, or left. Instead of sensory words, refer to the element by its label or tooltip, or use these alternatives: (examples follow).
- Avoid inline parens.

Reply:
Done